### PR TITLE
Deprecate Hawkular Provider

### PR DESCRIPTION
**Pull request by israel-hdez**
- Place deprecation note in blog post.
- Remove Hawkular as a supported platform in MiQ landpage
- Remove unused assets

### CODE REVIEW

**Comment by israel-hdez** [comment]
ping @abonas

**Comment by abonas** [comment]
LGTM
@JPrause

**Comment by JPrause** [comment]
@cybette for your review.

File: site/_posts/2017-10-20-hawkular-alerts-in-miq.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ published: true
 tags: tutorials middleware
 ---
```
Comment (cybette) [inline site/_posts/2017-10-20-hawkular-alerts-in-miq.md]: Can you please remove the middleware tag? We're keeping the blog categories pretty broad for now, and this is the only post in the middleware category, and there probably won't be many more after this. Thanks!

Comment (israel-hdez) [reply site/_posts/2017-10-20-hawkular-alerts-in-miq.md]: It's removed now.

**Pull request merged**